<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<!--Converted with LaTeX2HTML 98.1p1 release (March 2nd, 1998)
originally by Nikos Drakos (nikos@cbl.leeds.ac.uk), CBLU, University of Leeds
* revised and updated by:  Marcus Hennecke, Ross Moore, Herb Swan
* with significant contributions from:
  Jens Lippmann, Marek Rouchal, Martin Wilck and others -->
<HTML>
<HEAD>
<TITLE>No Title</TITLE>
<META NAME="description" CONTENT="No Title">
<META NAME="keywords" CONTENT="sunprop">
<META NAME="resource-type" CONTENT="document">
<META NAME="distribution" CONTENT="global">
<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=iso-8859-1">
<LINK REL="STYLESHEET" HREF="sunprop.css">
<LINK REL="next" HREF="node1.html">
</HEAD>
<BODY >
<!--Navigation Panel-->
<A NAME="tex2html1"
 HREF="node1.html">
<IMG WIDTH="37" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="next" SRC="next_motif.gif"></A> 
<IMG WIDTH="26" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="up" SRC="up_motif_gr.gif"> 
<IMG WIDTH="63" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="previous" SRC="previous_motif_gr.gif">   
<BR>
<B> Next:</B> <A NAME="tex2html2"
 HREF="node1.html">Bibliography</A>
<BR>
<BR>
<!--End of Navigation Panel-->

<P>

<P>

<B>PROPOSAL TITLE: 
Spectroscopical and theoretical studies of interactions
between components of proteins and nucleic acids</B>

<P>
<BR>

<P>

<B>Abstract</B>

<P>
The fundamental properties of the nucleus of a living cell are
determined by the chromosomes containing the genes, and these consists
essentially of desoxyribonucleic acid (DNA) which  is the carrier
of the genetic information characteristic for the species. Many
biological and medical problems are directly related to the properties of
DNA and, since they govern all the duplication processes and protein
synthesis, they are of essential importance in the processes related to
normal growth and aging, as well as in the development of tumors.
In this collaborative research effort, we propose to combine 
high-resolution IR-matrix isolation, UV,
Rydberg electron transfer and mass spectroscopy
methods
with methods of
computational quantum chemistry and molecular modeling
to study interactions of the
components of DNA and proteins. In particular the investigations will
concern hydrogen-bonding, stacked and other types of non-bonding
interactions
in dimers and larger complexes of nucleic acid
bases and in complexes of nucleic acid bases with 
amino acids and smaller peptides. We will also investigate
how these interactions are influenced and altered by attachment of excess
electrons
and by electronic excitations of the interacting systems.
These factors, are related to increasing exposure of life on earth to
UV-irradiation due to depletion of the protective ozone layer. The
chemical consequences of the excessive UV radiation on the nucleic acid -
protein interaction can be profound. These include cell death,
mutation and other lethal transformations. Our study will reveal the
chemistry of these processes at the molecular level and, with the
assistance of the theoretical calculations, will allow to elucidate the
mechanisms of the damaging transformations and 
will allow to suggest possible
damage-preventing steps.
Only recently due to new advances in the high-resolution spectroscopical
methods and due to increasing capabilities of computer hardware and
software it has become possible to employ sophisticated physical
chemistry methods to study structure and chemistry of molecular systems
relevant to the human biology. Here we propose an
interdisciplinary collaborative 
research program which uses a diverse pallet of 
spectroscopical and computational methods 
to study the elemental interactions contributing
to the recognition processes involving DNA and proteins.
The team of researchers involved in the project include 
Prof.L. Adamowicz's computational chemistry group at the Department 
of Chemistry, University of
Arizona, U.S.A., Prof. G. Maes' matrix-isolation IR group at
the Chemistry Department, University of Leuven, Belgium,
Prof. Schermann's Rydberg electron transfer and mass spectroscopy
group at the
Laser Laboratory, the University of Northern Paris, France,
the computational
chemistry group of Prof. Duran-Portas at the University 
of Girona, Spain, and the matrix-isolation IR spectroscopy
group at the Prof. R. Fausto, Department of Chemistry,
University of Coimbra, Portugal.
The principle applicant (P.A.) has collaborated with all the members
of the team in the past and the results of the work have been
published in several papers. 
Now, P.A. proposes to form and broader research alliance to
investigate and determine the elemental molecular mechanisms
which contribute to the recognition and regulatory functions
of DNA and proteins.   

<P>

<P>

<B>1. OVERVIEW OF PROPOSED RESEARCH</B>

<P>
This proposal describes an international effort in basis research
on the complex molecular mechanism on the recognition and
regulatory functions of nucleic acids and proteins, which are
based on their interactions. 
In the proposed investigations we will use an 
interdisciplinary approach, which combines methods of 
high resolution spectroscopy with computational methods
of quantum chemistry and molecular modeling.
Each group participating in the study has developed a 
unique technique to study structures and properties of 
biological molecular systems. Only the union of these 
techniques will allow us to conduct a comprehensive 
investigation of the DNA-protein recognition
phenomenon at the molecular level.
The proposed research is a new undertaking for all the 
members of the team, and the requested funding is in now
way intended to be used to support ongoing domestic 
programs.

<P>
The idea of assembling a research team, which involvs both
theoretical and computational chemists and experimental 
spectroscopiests, has its roots in bilateral collaborations,
which Adamowicz's group has established over the recent
years with each of the
groups participating in the present project. The modern basic
chemical research has witnessed an increasing number of projects,
where theoretical and experimental chemists join forces
in studies of fundamental problems at the molecular level.
The members of this team have demonstrated the advantages
of using the combined theoretical-experimental approach in
a number of bilateral studies on molecular systems which are closely 
related to the systems considered in the present proposal (see the
publication lists
of the team members). Particularly, the collaborative work 
of Maes's and Adamowicz's groups on nucleic acid complexes with water,
the work of Schermann's and Adamowicz's group on electron
attachment to nucleic acid bases and related systems, and the 
work of Fausto's and Adamowicz's group on conformational behavior of
amino acids should be mentioned as examples of studies where
an experimental--theoretical approach has led to important new
results.

<P>
The groups participating in the project use the following
methods in their investigations:

<P>
<UL>
<P>
<LI>
Adamowicz's theoretical and computational chemistry group
has developed, implemented and applied <I>ab initio</I> and
semiempirical methods to study a wide range of systems
and their chemical and physical properties. In particular,
the work has concerned IR, Raman and NMR spectra of larger molecules and
complexes, electron attachment and electron transfer 
processes, electronic excitations and UV-induced transformations
in biomolecules. New methods based on multi-reference
coupled cluster approach have been developed to study
electron excitations and structural transformations of molecules.
Also, new techniques have been implemented to deal with
anharminicity of IR vibrations.
These new techniques has been applied in analyses of experimental
IR matrix isolation (MI) spectra of various biological molecules
including nucleic acid bases, amino acids, and their complexes.
Also recently, in collaboration with Dr. S. Stepanian and
his group in Kharkov, Ukraine, Adamowicz's group has performed extended
studies of the conformational behavior of the matrix-isolated
simple amino acids. The goal of this work was to determine the
structural flexibility of the amino acid molecules, which
is an important factor in determining the propensities of
amino acids to form H-bonding complexes with nucleic acid bases.
The Adamowicz-Stepanian team has also recently completed
two studies where theoretical calculations and mass spectroscopy 
were used to elucidate the thermodynamics of uracil and cytosine
(and their methylated forms) complexes with acylamide, which are
models for NAB-AA interactions. 

<P>
<LI>
Maes' experimental group has assembled a unique experimental
setup to study molecular complexes involving biomolecules
using MI IR spectroscopy.
Their studies has concerned complexes of nucleic acid bases
with water molecules and other simple closed-shell molecular
systems. The experimental data analyzed with the assistance
of the theoretically predicted spectra allows them to assign
the spectral bends to specific structural isomers and to determine
their relative concentrations in the isolated stage.
Meas' group will also bring to the present project their
expertise in IR studies of photoproducts of matrix-deposited
biomolecules, which they have conducted in collaboration
with Dr. M. Nowak group at the Institute of Physical Chemistry,
In Warsaw, Poland. 

<P>
<LI>
Schermann's group has developed and used a unique experimental
technique based on the Rydberg electron transfer (RET) spectroscopy 
and the mass spectroscopy to study anions of biomolecules
and their clusters. The technique allows studies of both
dipole-bound and covalently-bound
anions and determination of their
electron detachment energies. It also allows with the 
assistance of the
theoretical calculations to discriminate
between different structural isomers of the anions.

<P>
<LI>
Fausto's group has established a laboratory where a number of
spetroscopical methods, including IR and Raman spectroscopies,
are used to study biological systems. Particularly, their
recent studies of amino acids and related systems are most
closely related to the aims of the present proposal.

<P>
<LI>
Duran's computational group has establish a leading program
is studies of molecular recognition phenomenon. They also
have investigated intermolecular and intramolecular proton
transfer processes in ground and excited states. They have
mastered the use of wide range of quantum mechanical methods
to study non-equilibrium behavior of biological molecules.
There role in the project will be to study through
quantum mechanical calculations how UV-radiation
affects the H-bonding NAB-AA interactions 
and to determine photochemical transformations which
may occur in the UV-irradiated NAB-AA complexes. 

<P>
</UL> 

<P>
Each of the groups has performed numerous
studies of molecular systems, which have a biological relevance.
The complexity of the interactions between molecules of
DNA bases and amino acids, which is the aim of the present
project will require that the sophistication level of both
the theoretical calculations and the experiments is elevated
to a higher level. The mutual collaboration is essential
for this effort to succeed.

<P>
There propose work will address the following problems.
The key question in the complex
process of the protein-DNA recognition
is how to predict the specific target sites in DNA bases and base
pairs
recognized by certain amino acids and their sequences forming a
protein fragment.
The redundancy and conformational variability in the
interactions between amino acids and bases will make it
difficult to establish specificity patters for
particular base pair and amino acids.
Due to their
structural flexibility, amino acids with many different geometries
can form similar types of interactions with bases.
Simple structural rules are not expected to 
be sufficient to accurately
predict the specific target recognition patterns
but certain definite preferences can be established.
The search for these preferences will be conducted as a joint
effort between Adamowicz's group and the matrix isolation
IR groups of Maes and Fausto. The theoretical calculations
will allow to establish thermodynamic relations between
different configurations. Fausto group will also employ the Raman
spectroscopy,
which will be complementary to the IR studies.
The effect of electron attachment to NAB's, AA's and their
complexes will be studied in Schermann's laboratory using the
RET spectroscopy and in Maes' laboratory using IR MI 
spectroscopy, as well as using QM calculations in 
Adamowicz's group. Photochemistry of NAB-AA systems will
be investigated using QM calculations by Duran's group
and experimentally elucidated by Maes' group in collaboration
with Nowak's group in Warsaw.

<P>
Prior to preparation of this proposal the members
of the team have established dual contacts through
visits and other means including frequent electronic
communications. These modes of collaboration will be also 
used in the future collaboration. An important new component
of the collaboration will be yearly meetings of the leaders of
all the participating groups, where the project progress
will be assessed and plans for future studies will be made.
Being university-based, the primary goal of each group in the team
is to educate students
and to prepare them to become leaders in research, education
and industry. This project will provide a unique opportunity
for student exchanges between the groups. Their exposure 
in early stages of their carriers to
the diversity of research methods, which this team represents,
is an essential element of the present project.
Since this research effort includes many tasks, the 
coordination role of the principal applicant (Adamowicz)
will be critical to the success of the project. Since PA has
collaborated with all the team groups in the past,
the team members are convinced that he will provide an
effective leadership in the project.

<P>

<P>

<B>PROPOSED RESEARCH</B>

<P>
Gene expression in higher organisms is controlled
by a wide variety of regulatory proteins, which
bind to specific sites of DNA.
The growing protein structural databases provides
a rich source of information about the interactions
between amino acids (AA) and base pairs at the molecular level.
Analyses of the databases have revealed several classes
of DNA-binding proteins with distinct binding motifs
and a variety of interactions between proteins and DNA.
These motifs include helix-turn-helix structural sequence, 
which occurs to achieve a sequence specific fit of 
amino acids with DNA base pairs. The direct reading of DNA base
sequence by amino acids plays an important role
in the recognition process. 
Seeman <I>et al.</I>[<A
 HREF="node1.html#p1">1</A>]
It was suggested that formation of double H-bond bridges
(so called ``point contacts") between amino acids
and base pairs are required for them to discriminate
between different bases. This point is supported
by frequent occurance of such interactions as Asn-A
and Lys-G in the resolved DNA-protein structures.
However, the structural analysis also show
a considerable degree of redundancy  in the specific
interactions between amino acids and nucleic acid bases (NAB).
[<A
 HREF="node1.html#p2">2</A>]
Thus, there is more flexibility in the interactions
of AA and NAB than there is in the interactions of NAB pairs
[<A
 HREF="node1.html#p3">3</A>]
allowing for more structural motifs to occur.
As it seems, 
due to the structural flexibility of amino acids, 
one can identify not
definite NAB-AA pairs but a group
of pairs interacting in similar ways.
[<A
 HREF="node1.html#p4">4</A>,<A
 HREF="node1.html#p5">5</A>]
The conformational flexibility of protein and to a lesser
extent the structural flexibility of DNA is an
important factor determining structural NAB-AA matches.
Flexibility is, therefore, a key feature which affects the
NAB-AA binding activity and target recognition patterns. 
[<A
 HREF="node1.html#p11">6</A>]  
Our study will address this phenomenon.

<P>
NAB-AA Binding-site preferences, which will be 
investigated in this work, can be also
derived from DNA libraries[<A
 HREF="node1.html#p17">7</A>]
and from protein-DNA
structural database.[<A
 HREF="node1.html#p22">11</A>]
Since redundancy and conformational variability occur in the
interactions between amino acids and bases it is
difficult to establish the specificity patters for
particular base pair and amino acids and
simple rule-based methods are not expected to sufficient to accurately
predict
of DNA sequence recognition by proteins.
However, in spite of the absence of simple codelike
correspondence between bases and amino acids, the interaction 
propensities exist between these systems.[<A
 HREF="node1.html#pp">12</A>]
The elucidation of these propensities will be the primary
goal of the proposed work.

<P>
The interactions of proteins with nucleic acids involve the 
same intermolecular forces (hydrogen-bond, charges, dipoles,
and solvent-driven interactions) and the same relatively simple
groups of atoms that are involved in the interactions of these
macromolecules with small molecules. The interactions of these
types can exhibit significant overall specificity because both
the proteins and the nucleic acids are folded into highly specific
conformations allowing the functional groups to achieve
positional and directional specificity.
The conformational basis of the specificity of the peptide-nucleic
acid interactions makes even small changes in the conformation of
either component become a destabilizing factor in the interactions.
There can be several factors causing such changes including
change of the tautomerization due to inter- or intra-molecular
proton transfer, electronic excitations and electron attachment.
These destabilizing factors will be investigated in this work.

<P>
The primary source of interactional specificity between proteins and
nucleic acids is hydrogen bonding. Hydrogen bonds exhibit strongly
preferred lengths and angular direction, and a distortion of these
parameters can be energetically costly. 
Almost all AA and NAB 
residues carry one or more hydrogen-bond donors or acceptors.
Thus, just as for the stability of the Watson-Crick canonical base 
pairs relative to the many other hydrogen-bonded base pairs 
possible, it is not so much that the correct structures are 
stable as that the incorrect structures are unstable.
In addition to the hydrogen bonding stacking interactions are 
important factors in the structural stability of some AA-NAB 
conformations. Some of the amino residues in
proteins (Tyr, Trp, His, Phe) have appropriate planar configurations 
and could be involved in stacking interactions (intercalation)
with NAB's. Such interactions have been implicated
both in complexes of nucleic acids and oligopeptides[<A
 HREF="node1.html#b1">13</A>]
and in complexes of nucleic acids and proteins.[<A
 HREF="node1.html#b2">14</A>]
Specific complex formation with protein components probably
involves two or more specific hydrogen-bonding interactions
per base, just as it does for base-base interactions.
The interplay between the H-bonding and stacked interaction
in NAB-AA complexes will be investigated in this work.
It is clear that a more elaborate approach to understanding NAB-AA
specificity based on combined structure and thermodynamics arguments
has to be developed and this work addresses this need.

<P>
The fundamental question in protein-nucleic acid recognition is 
whether the specificity exists in the local NAB-AA interactions,
or if it
is fully determined by the spatial structure of
nonspecific elementary interactions. It seems 
that increasing evidence has been accumulated showing that
that the specificity is based on both the  "point" interactions between
chemical groups of the two interacting
systems and on the general structural complementarity
of interacting regions of the two macromolecules.
In the present work we will investigate the former. 
In the "point" interactions the carboxylate amino acid groups 
(<I>e.g</I>, Glu and Asp) play an important role among the 
amino acid side chains. 
The carboxylate groups of Asp and Gly, which can often be found in
protein-nucleic  acid binding sites, are one of the most active groups of
side chains at a site of complex formation. This is due to the presence of
a proton donor and a proton acceptor. The selective recognition of nucleic
acid bases by specific side chain amino acids provides a important
motivation
for the proposed investigation of this phenomenon at the molecular level
using physical chemistry methods.

<P>
Since the vast majority of the genom comprises non-site
DNA sequences, and  since site-specific DNA-binding 
proteins still have a weak affinity for the non-site DNA,
the protein must display a much higher binding affinity for its
own site(s) than for non-site DNA in order for the
regulatory system to work.[<A
 HREF="node1.html#s1">23</A>]
Several DNA-binding motifs[<A
 HREF="node1.html#s1">23</A>,<A
 HREF="node1.html#s2">24</A>,<A
 HREF="node1.html#s3">25</A>,<A
 HREF="node1.html#s4">26</A>,<A
 HREF="node1.html#s5">27</A>]
have been studied primarly using
crystal structures of protein-DNA complexes.
to determine whether a ``DNA recognition code", which
sets out rules for amino acid-NAB pair interactions is similar
to the deterministic genetic code. The work proposed
here may provide important answers to these questions, which
will be relevant to
current large-scale sequencing projects, as
well as to the effort of monitoring gene expression processes.

<P>
The relevance of the proposed research also extends to
understanding of the thermodynamics of drug binding to DNA. 
Thermodynamics offer key
insight into
the molecular forces that drive NAB-AA complex formation that cannot be
obtained
by structural or computational studies alone.
Obligatory conformational changes in the DNA to accomodate intercalators
and the loss of translational and rotational freedom upon complex
formation both present unfavorable free energy barriers for binding. Such
barriers must be overcome by favorable free energy contributions from the
hydrophobic transfer of ligand from solution into the binding site, and
molecular interactions that form within the binding site.[<A
 HREF="node1.html#e1">28</A>]
Interest in the binding of small molecules including amino acids
to DNA arises from both practical and fundamental reasons. On the
practical level, many small molecules that bind to DNA are effective
pharmaceutical agents, especially in cancer chemotherapy. Understanding
the DNA binding to such drags is essential in understanding their mode of
action and for the development of design principles to guide the synthesis
of new, improved compounds with enhanced or more selective activity. On
the fundamental level small molecule binding to DNA offers a simple,
well-defined system for understanding general principles that contribute
to the free energy of biomolecular complex formation. 

<P>
There are some specific types of interactions in NAB-AA
complexes, which are 
particularly interesting. Among them 
the CH<IMG
 WIDTH="29" HEIGHT="18" ALIGN="BOTTOM" BORDER="0"
 SRC="img2.gif"
 ALT="$\dots$">O hydrogen-bonds have recently gained interest due
to a number of reports indicating their significance in stabilizing
nucleic acid and protein structures.[<A
 HREF="node1.html#a1">31</A>]
Also, the aromatic hydrogen bond, <I>i.e.</I> the interaction
between the <IMG
 WIDTH="18" HEIGHT="18" ALIGN="BOTTOM" BORDER="0"
 SRC="img3.gif"
 ALT="$\pi$">-electron cloud of an aromatic ring and a
hydrogen-bond donor, has been considered for it ability to substitute for
a conventional hydrogen bond in sequence-specific protein-DNA
interactions.[<A
 HREF="node1.html#a2">32</A>]
Since these interactions, in addition to the classical hydrogen
bonding interactions between amino acids of the protein and the nucleic
acid bases of DNA, provide a more complete molecular picture of the
amino acid-base recognition, their structures and energetics
will be investigated in this work.

<P>
<BR>
<BR>

<P>
<B>(b) Objectives</B>

<P>
The objectives of the proposed research are:

<P>
<UL>
<P>
<LI>
Determination of fundamental structural and thermodynamical
properties of the NAB-AA interactions at the molecular level
with the use of
high-resolution spectroscopical methods and quantum mechanical
calculations.

<P>
<LI>
Esablishing relations between elemental NAB-AA bonding properties
and the recognition functions of these complexes.

<P>
<LI>
Elucidate how an excessive UV-exposure and attachment
of excess electrons can affect  
the NAB-AA interactions and alter their structural
features.

<P>
</UL>
<P>
To meet the above-defined objectives the research 
coordinated by PA will be
conducted simultaneously in all the participating research
groups. The allocation of the specific research
projects
to each team member will be the following. Fausto's and Maes'
groups will use their matrix-isolation IR/Raman instruments to 
conduct structural analysis of the NAB-AA complexes. The
analysis of the spectra will be assisted by the calculations
performed in the Adamowicz's group. This group will also 
conduct calculations on anions of NAB and AA complexes.
This work will parallel Schermann's RET experiments on these
systems. Duran's theoretical analysis will concern static and dynamic
analysis of processes induced in NAB-AA complexes by UV-radiation.
Their calculations will be coupled with matrix-isolation
analysis of photoproducts performed by Maes in collaboration with
Nowak's group in Warsaw. Some analysis of the energetics
of the NAB-AA complexes will be done by Adamowicz's collaborator
Stepanian and his group in Kharkov. Stepanian will visit 
Adamowicz's group for extended periods of time to assist with the
calculations. The results of the research will be shared
with all team members through the specially designed web site.
Through this medium all members will directly participate in the 
analysis of the results and in advancing the project goals.
Since the expertise of each team member is different,
this instant access to the results and 
participating in their analysis
will enforce the collaboration.

<P>
The NAB-AA systems which we intend to study include AA's 
and related model systems with functional groups, which
are particularly critical to the 
DNA-protein recognition motifs, <I>i.e.</I>
-OH (serine and hydroxyproline), -CONH<SUB>2</SUB> (asparagine and
glutamine), -COOH (aspartic and glutamic acids),
-SH (cysteine), -C(NH<SUB>2</SUB>)<SUB>2</SUB> (arginine) and <IMG
 WIDTH="18" HEIGHT="18" ALIGN="BOTTOM" BORDER="0"
 SRC="img3.gif"
 ALT="$\pi$">-electrons
in aromatic ring (phenylalanine and tryptophan). 
The NABs considered will include all DNA/RNA bases as
well as their methylated forms. The possible appearance
of "rare" NAB and AA tautomers and rotamers in NAB-AA
complexes will be considered. We will also consider
canonical and non-canonical NAB pairs and their interactions
with AA's and the role of water.

<P>
The following is a detail description of the research
project of the each collaborating team member:

<P>
<UL>
<P>
<LI>
Maes' and Fausto's groups will study fundamental 
structural properties of NAB-AA dimers using matrix-isolation IR and
Raman spectroscopy methods.
The goal of the study will be to determine the
bonding motifs which occur in these systems.
Even the simplest of these complexes can adapt
several conformations corresponding to different geometrical
isomers, thus greatly enriching their dynamic behavior in
the biological environment. 
The hydrogen-bond cooperativity effects
and hydrogen transfer behavior
leading to NAB tautomerism are important elements and
they will be elucidated in the investigations. 
Particularly the phenomenon of increasing mobility of the hydrogens
and higher propensity towards tautomerization
in the presence of water will be
studied.
The main aim of the study will be to establish
and explain at the molecular level
the specificity
of the recognition of NAB's by AA's
and to determine the role of different atomic groups
in the recognition functions.

<P>
<LI>
The above experimental work will be coupled with the
the quantum-mechanical calculations performed in
Adamowicz's group. They will help assigning the 
experimental spectra and explain based on structural
and thermodynamical considerations the observed 
NAB-AA interaction patterns.
As it has been in the past, this work will
stimulate development
of new theoretical methodology
for more accurate theoretical simulation of IR and
Raman spectra
of molecular complexes in matrix environments.
The theory development and 
calculations will also be done to assist the
analysis of Schermann's experiments on NAB complexes
with excess electrons.

<P>
<LI>
In the gas-phase experiments of Schermann 
the molecular or cluster anions are generated
through electron transfer collisions between the neutral
systems and laser-excited Rydberg atoms and
characterized <I>via</I> the dependence of the
anion formation rates on the Rydberg electron quantum numbers.
Both covalently-bound and dipole-bound anions will be
investigated.
As already established the
excess electron attachment may have profound 
structural consequences in complexes involving NAB's.
Free electrons are produced in secondary reactions
following interaction of living cells with ionizing
radiation. This investigation will reveal the
destabilizing effect of excess electrons on NAB dimers and
their complexes with water and AA's, 
and help determine molecular substitutions,
which can reduce
the susceptibility of DNA bases to the damage caused
by free-electrons (Schermann and Adamowicz recently
found that methylation has such a role).

<P>
<LI>
Duran's group in collaboration with Adamowicz's group will perform
theoretical investigations
of the outcome of the event that starts with promoting the 
components of NAB-AA complexes to excited electronic states by 
UV-radiation. The excited state proton transfer reaction is one
possible outcome, but many other transformations are also 
possible. This include conformational isomerization, ring
opening, aggregation, <I>etc.</I> Due to extended networks of hydrogen
bonds in the DNA-protein interactions, hydrogen migration
should occur, particularly when the hydrogen mobility is enhanced
by electronic excitations. 
The theoretical work will be coupled with the matrix-isolation
experiments of Maes and Nowak.

<P>
</UL>
<P>

<P>

<B>References:</B>

<P>
<BR><HR>
<!--Table of Child-Links-->
<A NAME="CHILD_LINKS">&#160;</A>
<UL>
<LI><A NAME="tex2html3"
 HREF="node1.html">Bibliography</A>
<LI><A NAME="tex2html4"
 HREF="node2.html">About this document ... </A>
</UL>
<!--End of Table of Child-Links-->
<HR>
<!--Navigation Panel-->
<A NAME="tex2html1"
 HREF="node1.html">
<IMG WIDTH="37" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="next" SRC="next_motif.gif"></A> 
<IMG WIDTH="26" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="up" SRC="up_motif_gr.gif"> 
<IMG WIDTH="63" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="previous" SRC="previous_motif_gr.gif">   
<BR>
<B> Next:</B> <A NAME="tex2html2"
 HREF="node1.html">Bibliography</A>
<!--End of Navigation Panel-->
<ADDRESS>
<I></I>
<BR><I>1999-06-22</I>
</ADDRESS>
</BODY>
</HTML>
